PersonalPortfolio/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Marcelo Garcia Personal Portfolio</title>
+</head>
+<body>
+<!--header section  -->
+<header>
+    <div class="nav-menu">
+        <a href="#">Marcelo Garcia</a>
+    <nav>
+        <img class="hamburger-btn" src="/PersonalPortfolio/imagenes/icon - menu.png" alt="menu-button">
+    </nav>
+    </div>
+</header>
+<!--header -->
+<section id="hero">
+    <div class="headlines">
+        <div class="headline-title">
+            <h1>I’m Marcelo
+                Glad to see you!</h1>
+        </div>
+        <div class="hero-text">
+            <p>I’m a software developer! I can help you 
+                build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need 
+                coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect">
+            <span>LET’S CONNECT</span>
+        </div>
+        <ul class="social-icons">
+            <li><img src="/PersonalPortfolio/imagenes/twiter.png" alt="twiter"></li>
+            <li><img src="/PersonalPortfolio/imagenes/Linkedin icon.png" alt="Linkedin"></li>
+            <li><img src="/PersonalPortfolio/imagenes/git.png" alt="git-ic"></li>
+            <li><img src="/PersonalPortfolio/imagenes/chill.png" alt="chill-ico"></li>
+        </ul>

```

Comment:
- [ ] As the previous reviewer requested you to use anchor tags around your imag icons. Kindly use them as shown below template :arrow_down:

```
<ul>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
  <li><a><img /></a></li>
</ul>
```